common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -209,7 +214,7 @@ def get_real_social_auth_object(request):
     if running_pipeline and 'social' in running_pipeline['kwargs']:
         social = running_pipeline['kwargs']['social']
         if isinstance(social, dict):
-            social = UserSocialAuth.objects.get(uid=social.get('uid', ''))
+            social = social_django.models.UserSocialAuth.objects.get(uid=social.get('uid', ''))
```

Comment:
I think getting rid of this function (if it really is only being used in that 1 place) and replacing it's usage with `partial_load` sounds much better; less code to maintain and overall more correct, as you noted. I'll take note of this in the edx-enterprise PR.